recipes/openssl/openssl_1.0.2g.oe

```diff
@@ -1,8 +1,5 @@
 require openssl.inc
 
-# Parallel make is simply broken for now.
-PARALLEL_MAKE = ""
```

Comment:
Do you have anything concrete knowledge about parallel build fixes?  Or just re-enabling so we can find out?  Would be nice to know if we should be particularly on the look for such issues...